commands/download.js

```diff
@@ -1,14 +1,10 @@
 var chalk = require('chalk')
 var prettyBytes = require('pretty-bytes')
-var Dat = require('dat-js')
 var logger = require('status-logger')
 var speedometer = require('speedometer')
 var ui = require('../lib/ui')
 
-module.exports = function (args) {
-  if (args && args.exit) args.upload = false
```

Comment:
yeah i think its fine to get rid of.

also i'm not sure what `args.upload = false` was doing or how that was related